easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1508,6 +1508,27 @@ def det_location_for(path, target_dir, soft_name, target_file):
     return target_path
 
 
+def cleanup_easyconfigs(paths):
+    """
+    Clean up easyconfigs (in place) by filtering out comments/buildstats included by EasyBuild in archived easyconfigs
+    (cfr. FileRepository.add_easyconfig in easybuild.tools.repository.filerepo)
+
+    :param paths: list of paths to easyconfigs to clean up
+    """
+    regexs = [
+        re.compile(r"^# Built with EasyBuild.*\n", re.M),
```

Comment:
In this case `re.M` isn't required, I think you're right.

Can you clarify the question on the `\n`? If you're asking if that's deleted too, yes, it is, and it should be, otherwise you'll end up with a blank line?